prdoc/pr_5201.prdoc

```diff
@@ -0,0 +1,20 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Snowbridge free consensus updates
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Allow free consensus updates to the Snowbridge Ethereum client if the headers are more than a certain
+      number of headers apart.
```

Comment:
How does this impact the devs? Is it cheaper to run relayers now?
Does this affect the user somehow? E.g. cheaper to use the bridge?

Reply:
https://github.com/paritytech/polkadot-sdk/pull/5201/commits/ab8739cff56a96ad838020a451d9809b7f617d15#diff-cfa518fee4956eaf8afc94f3fad7a131b1c1474c2be560798ba1c5cec04f57dcR10